integration/js/utils/SdkBaseTest.js

```diff
@@ -165,11 +165,11 @@ class SdkBaseTest extends KiteBaseTest {
         console.log(e);
         await this.updateSeleniumTestResult(false);
         await this.quitSeleniumSessions();
-        await emitMetric('Common', this.capabilities, 'SeleniumInit', 0);
+        await emitMetric('Common', this.capabilities, 'SeleniumInit', 0, this.payload?.isPreviousMajorVersion);
```

Comment:
Shall we use a global variable `this.isPreviousMajorVersion` to replace `this.paylaod?.isPreviousMajorVersion`?

Reply:
We can use a global variable for `isPreviousMajorVersion` but there is no real benefit of doing it. 

This variable is being sent to KITE via the `payload` property. Some tests will have this property set to true while others will not have this property. We can make it a global property and set it to false by default and overwrite it whenever we do get it from the payload but not sure if its worth it.

Reply:
I think currently we are assigning the payload data to global variables in the constructor. If there's no specific reason of not doing this, I would prefer to follow the convention. There's no specific benefit of doing this in this case. But it is a good practice to follow. The constructor could helps ensure the variable is in the good states before we use it. 
Although `this.payload?.isPreviousMajorVersion)` could handle the same logic here, but constructor could better manager the states.

https://github.com/aws/amazon-chime-sdk-js/blob/fca27b4f6e45160b7fbbde9115b16c826ed472bf/integration/js/utils/SdkBaseTest.js#L19-L25